vm/dependencies.go

```diff
@@ -51,7 +46,6 @@ type Config struct {
 	ProcessingBuildSkip              int             `json:"processingBuildSkip"`
 	TargetGossipDuration             time.Duration   `json:"targetGossipDuration"`
 	BlockCompactionFrequency         int             `json:"blockCompactionFrequency"`
-	EnableJSONRPCStateHandler        bool            `json:"enableJSONRPCStateHandler"`
```

Comment:
In the future, we should require an e2e test before adding a piece of functionality like this, but for now, should we add a test or GitHub issue to add a test, so that we don't leave this as dead code?

Reply:
Added an issue: https://github.com/ava-labs/hypersdk/issues/1292